src/script/lua_api/l_object.cpp

```diff
@@ -2313,8 +2301,7 @@ int ObjectRef::l_set_lighting(lua_State *L)
 	lua_pop(L, -1);
 
 	getServer(L)->setLighting(player, lighting);
-	lua_pushboolean(L, true);
-	return 1;
+	return 0;
```

Comment:
You need to update the documentation to remove "Returns true on success"

This is fine as set_lighting is new